SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+The FreeCAD project is a FOSS (Free and Open-Source Software) project that has a community of thousands of users and 
+hundreds of developers worldwide. We encourage responsible reporting of security vulnerabilities that may affect users
+of this software, and will endeavor to address these vulnerabilities when they are discovered.
+
+## Bounties
+
+We do not pay bounties to identify bugs, and we feel that it sets a dangerous precedent.  It encourages people to
+withhold information that is valuable to all in order to receive a personal financial benefit.  This runs against the
+spirit and history of the FreeCAD project. If you discover a vulnerability that affects a part of the FreeCAD project
+(either directly in FreeCAD, in a library it depends on, or in any of the various other sub-projects such as our
+website, forums, etc.) we ask you to join the large community of volunteer contributors and file a report about the
+issue.
+
+## Supported Versions
+
+FreeCAD implements security fixes to the current release: currently 0.20.x, and to the current development release, currently called 0.21.
+
+| Version | Supported          |
+|---------| ------------------ |
+| 0.21    | :white_check_mark: |
+| 0.20.x  | :white_check_mark: |
```

Comment:
This may sound like we support ALL `0.20.x`, as opposed to saying it's best to update to the highest available `x`. Are you sure that's how it works?

Reply:
I was cribbing off of a GitHub example, but you are right, I should use the specific version number.